.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can you remove the `.pytest_cache` entries from this commit please?

Reply:
Of course I can.

This is embarrassing. I usually check what I commit, but not this time :( .